torchgeo/datamodules/chabud.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""ChaBuD datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+import torch
+from einops import repeat
+
+from ..datasets import ChaBuD
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class ChaBuDDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the ChaBuD dataset.
+
+    Uses the train/val splits from the dataset
+
+    .. versionadded:: 0.5
+    """
+
+    # min/max values computed on train set using 2/98 percentiles
+    min = torch.tensor(
+        [0.0, 1.0, 73.0, 39.0, 46.0, 25.0, 26.0, 21.0, 17.0, 1.0, 20.0, 21.0]
+    )
+    max = torch.tensor(
+        [
+            1926.0,
+            2174.0,
+            2527.0,
+            2950.0,
+            3237.0,
+            3717.0,
+            4087.0,
+            4271.0,
+            4290.0,
+            4219.0,
+            4568.0,
+            3753.0,
+        ]
+    )
+
+    def __init__(
+        self, batch_size: int = 64, num_workers: int = 0, **kwargs: Any
+    ) -> None:
+        """Initialize a new ChaBuDDataModule instance.
+
+        Args:
+            batch_size: Size of each mini-batch.
+            num_workers: Number of workers for parallel data loading.
+            **kwargs: Additional keyword arguments passed to
+                :class:`~torchgeo.datasets.ChaBuD`.
+        """
+        super().__init__(ChaBuD, batch_size, num_workers, **kwargs)
+
+        self.bands = kwargs.get("bands", ChaBuD.rgb_bands)
+        band_indices = [ChaBuD.all_bands.index(b) for b in self.bands]
+        _min = self.min[band_indices]
```

Comment:
According to [PEP 8](https://peps.python.org/pep-0008/#descriptive-naming-styles) you should use a trailing underscore instead of a leading underscore to avoid conflicts with builtin functions. Also, you should use the same thing for the class attributes.